package.json

```diff
@@ -104,15 +105,16 @@
 			"test/*"
 		],
 		"timeout": "1m",
-		"typescript": {
-			"compile": "tsc",
-			"rewritePaths": {
-				"test/": "dist/test/"
```

Comment:
Is this not needed? `dist` is still referenced in `exports`.

Reply:
No, that config is just for tests. We now use `ts-node` instead which does this for us.